src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -48,6 +57,27 @@ export const selectPortfolioAccountBalances = (
   state: ReduxState,
 ): PortfolioAccountBalances['byId'] => state.portfolio.accountBalances.byId
 
+export const selectIsPortfolioLoaded = createSelector(
+  selectAccountSpecifiers,
+  selectPortfolioAssetIds,
+  (accountSpecifiers, portfolioAssetIds) => {
+    if (!accountSpecifiers.length) return false
+    /**
+     * for a given wallet - we can support 1 to n chains
+     * PortfolioContext ensures we will have a portfolioAssetId for each chain's fee asset
+     * until the portfolioAssetIds includes supported chains fee assets, it's not fully loaded
+     * the golf below ensures that's the case
+     */
+
+    return !size(
+      difference(
+        uniq(map(accountSpecifiers, flow([keys, head]))),
+        uniq(map(portfolioAssetIds, assetIdtoChainId)),
+      ),
+    )
```

Comment:
I'm not sure how hard selectors are to test, but I'd like to see some tests for this.  If it's hard, then refactor this logic into a function and test that function.

Reply:
tests added